### PR TITLE
updates

### PR DESCRIPTION
**Pull request by zotterb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,8 +13,8 @@ Create a file called `.env` with the following variables:
 
 ```
 API_TOKEN=567fde7b94ff711111111111111111111111
-API_HOST=https://api.sandbox.getcacheflow.com
-API_HOST_HEADER=bananastand.api.sandbox.getcacheflow.com
+API_HOST=https://api.sandbox.getcacheflow.com | https://api.getcacheflow.co
```
Comment (driedtoast) [inline README.md]: `.com`

File: src/api/types.ts [hunk 2]
```diff
@@ -1,398 +1,50 @@
...
-  
-    /** @format date-time */
-    createdAt?: string;
-    createdBy?: string;
-    contact?: Contact;
-    recipient?: Contact;
-    eventType?: "created" | "updated" | "activated" | "viewed" | "accepted" | "invited" | "cancelled";
-  }
-
-export interface Proposal {
```
Comment (driedtoast) [inline src/api/types.ts:353]: shouldn't we have some bits in here about this?

**Review by driedtoast — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zotterb) [reply README.md]: 👍

File: src/api/types.ts (hunk 2 above)
Comment (zotterb) [reply src/api/types.ts:353]: Yeah I'll add back things we want to expose over time.

**Pull request merged**